pynest/nest/tests/test_kinetics_multi_cmpt_neu.py

```diff
@@ -0,0 +1,389 @@
+# -*- coding: utf-8 -*-
+#
+# test_kinetics_multi_cmpt_neu.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+# This script tests the iaf_cond_alpha_mc_kinetics_neuron in NEST,
+# while comparing it with an implementation of the neuron model in
+# python using ode solver. The inputs to both neuron models are spikes
+# to an excitatory synapse at the distal compartment to trigger a calcium
+# spike. The membrane potential at the distal compartment of the models
+# are then compared.
+
+import numpy as np
+import pylab as pl
+import matplotlib.cm as cm
+import math
+import nest
+from scipy import *
+from scipy.integrate import ode
+import unittest
+
+HAVE_GSL = nest.sli_func("statusdict/have_gsl ::")
+
+
+@unittest.skipIf(not HAVE_GSL, 'GSL is not available')
```

Comment:
This decorator must be moved to the `KineticsMultiCmptNeuTestCase` class so that the test is skipped. Now, you just skip the definition of the `mc_neuron_ode` class.